services/terraformer/server/clusterBuilder/clusterBuilder.go

```diff
@@ -201,6 +208,12 @@ func (c ClusterBuilder) getNodes() []*pb.Node {
 	return oldNodes
 }
 
+// ReplaceAll is a wrapper function to replace all occurrence of a substring with another
+// substring. Returns modified string
+func (nd NodepoolsData) ReplaceAll(s, old, new string) string {
+	return strings.ReplaceAll(s, old, new)
+}
+
```

Comment:
This function should be part of `NodepoolsData` struct, move the function to this [file](https://github.com/Berops/claudie/blob/ef31f9951082f7d8b2f22df8694884306054f583/internal/templateUtils/templates.go#L1) and then add it [here](https://github.com/Berops/claudie/blob/ef31f9951082f7d8b2f22df8694884306054f583/internal/templateUtils/templates.go#L70) to call it from the templates

Reply:
Actually I think you can just add something like this to the FuncMap [here](https://github.com/Berops/claudie/blob/ef31f9951082f7d8b2f22df8694884306054f583/internal/templateUtils/templates.go#L71) and you shouldn't even need a function definition.
```go
"ReplaceAll": strings.ReplaceAll
```

Reply:
yeah i was not aware of the function map, is sounds like i dont need to define a special wrapper function, i will pass the `strings.ReplaceAll` in the function map